src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -46,8 +46,15 @@ var meterProvider = Sdk.CreateMeterProviderBuilder()
     .Build();
 ```
 
-See the [`TestMetrics.cs`](../../examples/Console/TestMetrics.cs) for
-runnable example.
+By default, `AddOtlpExporter()` pairs the OTLP MetricExporter with a
```

Comment:
Ideally we just document defaults in the "## Configuration" section for all 3 signals. But that can be a future re-factoring.

Reply:
Agree. There are some changes coming to logs and traces. Once those are done, we can update the configuration section. I just added this for now as we get common ques around how Otlp metric exporter is configured?